sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
+		return errors.New("Query contains unsafe character")
+	}
+	return m.SelectOne(holder, q, args...)
```

Comment:
Hrm, I'm not entirely convinced it would be that useful, we'd still need an internal `fmt.Sprintf` to concat the two forms, I think the only advantage is that we could run the `strings.ContainsAny` against only the field list, but that's not really that big of a win.